guide/creating-your-bot/command-deployment.md

```diff
@@ -1,12 +1,16 @@
 # Registering slash commands
 
 ::: tip
-This page assumes you use the same file structure as our [Slash commands](./slash-commands.md) section, and the provided are made to function with that setup. Please carefully read that section first so that you can understand the methods used in this section.
+For fully functional slash commands, there are three important pieces of code that need to be written. They are:
 
-If you already have slash commands set up and deployed for your application and want to learn how to respond to them, refer to the following section on [Command Response Methods](/slash-commands/response-methods.md).
-:::
+1. The [individual command files](slash-commands.html), containing their definitions and functionality.
```

Comment:
I see some irregularities across the guide, but I think we generally omit `.html` on internal links, or if necessary use `.md` https://github.com/discordjs/guide/blob/main/CONTRIBUTING.md#images-and-links